lib/PTML/Tag.cpp

```diff
@@ -0,0 +1,44 @@
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/PTML/Constants.h"
+#include "revng/PTML/IndentedOstream.h"
+#include "revng/PTML/Tag.h"
+
+namespace ptml {
+PTMLBuilder::PTMLBuilder(bool GenerateTagLessPTML) :
+  GenerateTagLessPTML(GenerateTagLessPTML) {
+}
+
+bool PTMLBuilder::isGenerateTagLessPTML() const {
+  return GenerateTagLessPTML;
+}
+
+ptml::Tag PTMLBuilder::getTag(llvm::StringRef Tag) const {
+  return ptml::Tag(Tag);
```

Comment:
This should check if the `PTMLBuilder` is tagless and emit an empty tag if that's the case.